RELEASE/scripts/autoscend.ash

```diff
@@ -5149,7 +5150,11 @@ void auto_begin()
 		use_familiar($familiar[none]);
 	}
 	dailyEvents();
-	while((my_adventures() > 1) && (my_inebriety() <= inebriety_limit()) && !(my_inebriety() == inebriety_limit() && my_familiar() == $familiar[Stooper]) && !get_property("kingLiberated").to_boolean() && doTasks())
+	
+	// the main loop of autoscend is doTasks() which is actually called as part of the while.
+	// keep the last 2 adventures in case we need to craft food/drink that costs 2 adv to make.
+	// Copiers and free fights also require at least 1 adventure left
+	while((my_adventures() > 2) && (my_inebriety() <= inebriety_limit()) && !(my_inebriety() == inebriety_limit() && my_familiar() == $familiar[Stooper]) && !get_property("kingLiberated").to_boolean() && doTasks())
```

Comment:
While I'm not against this per se, it seems a bit unnecessary to always reserve any adventures regardless of whether we have any free fights to use at the end of the day or not.

Reply:
It is also for the cooking. on my side account I don't have IOTMs to provide me with food, and as such need to resort to cooking lo mein, scrumdiddlyumptious noodles, and hi mein. As well as cocktailcrafting the upgraded drinks. All of them require 2 adventures to craft.
https://kol.coldfront.net/thekolwiki/index.php/Dry_noodle_guide
https://kol.coldfront.net/thekolwiki/index.php/Advanced_Cocktailcrafting#Upgraded_Recipe_Guide

Reply:
The drinks btw are more difficult to handle. You can check if stomach is full, but with drinks you have nightcap letting you drink beyond your liver limit

Reply:
Done. added new code.